docs/source/interfaces/Engineering Diffraction Test Guide.rst

```diff
@@ -156,6 +156,44 @@ Negative Testing Ideas
 - Change any unlocked dialog boxes whilst `Fit` runs
 
 
+.. _gsas-Engineering-Diffraction_test-ref:
+
+GSAS Fitting
```

Comment:
Is it worth matching the heading with the name of the tab in the GUI (as per the other sections/tabs)?